providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/process/storage/FlagStore.java

```diff
@@ -150,9 +150,9 @@ private void streamerListener(final QueueSource connector) throws InterruptedExc
         log.info("Shutting down store stream listener");
     }
 
-    private Structure parseSyncMetadata(GetMetadataResponse metadataResponse) {
+    private Structure parseSyncContextOrMetadata(Struct syncContext) {
```

Comment:
[nit] to future proof this a little, we should just rename it to:

```suggestion
    private Structure parseSyncContext(Struct syncContext) {
```